default.nix

```
@@ -13,6 +13,10 @@ let
     package = pkgs.web-security-tracker;
     module = import ./nix/web-security-tracker.nix;
 
+    manage = pkgs.writeScriptBin "manage" ''
+      ${python3}/bin/python ./src/website/manage.py $@
```

```suggestion
      ${python3}/bin/python ${toString ./src/website/manage.py} $@
```

This makes it independent of CWD